src/js/components/Articles/modules/ReactReader/style.js

```diff
@@ -0,0 +1,141 @@
+export default {
```

Comment:
why is this styling file a js file? and not a css type file?

Reply:
I got the code like that from github reactReader repo, we tried to convert the file in question but the way it is setup, the files is very integrated and can't be converted without having to re-write majority of the code. I hope the reason is sufficient.

Reply:
so instead of fixing it you are going to leave it wrong because it integrated in a lot of places???

Reply:
It's took us over an hour to actually see we messing it up when we tried to correct it, yes its not a good approach but for the fact that my peers are waiting for it I do not have enough time to restructure it accordingly.

Reply:
I finally got the package that have the reactereader but needed some completions using other parts of my previous one. This package replaced the modules folder (with EPubView and eReader) so I deleted it completely and installed the react-reader package and got it the work as it wasn't well explained

Reply:
Separated .scss files from .js files